### DISCUSSION TITLE
Not able to run with SCIPU_BACKEND

### DISCUSSION THREAD

**Discussion by sreenivasanm6** [cvxpy/cvxpy#1960] [General]
Tried using below line of code for solving the long compilation time issue, but getting below error.

AttributeError                            Traceback (most recent call last)
/var/folders/kb/dch64ql97jd6krk7snwwv1gc0000gn/T/ipykernel_647/2682614393.py in <module>
    898     prob = Problem(obj, constraints)
    899     print("I am here")
--> 900     prob.solve(verbose=True, solver =  cp.MOSEK, backend=cp.SCIPY_BACKEND)
    901 
    902     df2 = test_data.copy()

AttributeError: module 'cvxpy' has no attribute 'SCIPY_BACKEND'

Tried installing cvxpy from source and getting the above error. The version installed is 13.0 and using python 3.7.13

**Comment by phschiele** [comment]
Did you install from source in a fresh clone of CVXPY?
Can you post the output of:
```py
>>> import cvxpy as cp
>>> cp.__version__
```
It should look something like this: '1.3.0.dev0+472.56eb02d'

**Reply by sreenivasanm6** [reply]
Here is the output:

>>> import cvxpy as cp
>>> cp.__version__
'1.3.0.dev0+453.98fccd9'

**Reply by sreenivasanm6** [reply]
Let me know if there is any update on this please?

**Reply by sreenivasanm6** [reply]
Team, let me know is there any way to resolve this issue? Please note that this was tried from fresh clone source installation only.

**Reply by rileyjmurray** [reply]
Ping @phschiele

**Reply by phschiele** [reply]
@sreenivasanm6 Sorry for the delay in response, I had to wrap up a few things on my end.

You should use `cp.SCIPY_CANON_BACKEND`, not `cp.SCIPY_BACKEND`.
Was there any offending documentation that indicated otherwise?
